perf/istio-install/create_cluster.sh

```diff
@@ -20,15 +20,17 @@
 # get default cluster version for zone
 function default_cluster() {
   local zone=${1:?"zone required"}
-  local temp_ver=$(mktemp)
+  local temp_ver
+  temp_ver=$(mktemp)
```

Comment:
The reason this and other similar constructs were split in two is because

local foo=$(some command)

ends up ignoring any failures in $(some command). To fix, you need to separate variable declaration from command execution.